### DISCUSSION TITLE
Do bladebit created portable plots actually work?

### DISCUSSION THREAD

**Discussion by jayhohoho2019** [Chia-Network/bladebit#82] [Q&A] [answered]
Just wondering if there has been any observations/measurements there.  I'm referring to plots created with the -c flag.  None of mine (created with bladebit) has farmed a block whereas madmax portable and OG plots have, since Aug 15th.  All plots do pass **chia plots check**

**Comment by preutrn** [comment, marked as answer]
After the initial bug was fixed #7 portable plots are A-OK.

**Reply by jayhohoho2019** [reply]
Thanks. So from that thread if I pulled the code after July 21st I'd be fine?

**Reply by harold-b** [reply]
Correct. I'd recommend pulling and building from the latest changes to the master branch to be sure, as these are considered stable.

**Reply by jayhohoho2019** [reply]
Thanks.  I pulled on Aug 15th actually.  Any changes since then?

**Reply by harold-b** [reply]
No changes to master since Aug 8th so you should be good. Make sure your local commit hash matches the one pointed to by master: `f3fbfff43ce493ec9e02db6f72c3b44f656ef137`.

You can verify this in the command line by typing `git rev-parse HEAD`

**Reply by jayhohoho2019** [reply]
Thanks you. The hashes match.

**Reply by colbyboles** [reply]
Can confirm that I have won XCH solo farming these portable plots.
